test_schemas/en/test_new_routing_checkbox_contains.json

```diff
@@ -0,0 +1,107 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.2",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Test Routing Checkbox Contains In",
+    "theme": "default",
+    "description": "A test survey for routing based on in condition",
```

Comment:
Should emphasis the operator, as it is not clear. Same for the other schemas.

Reply:
Emphasis added, now operators are all capitalised. Thanks!